hasura/migrations/1615594547538_guild_add_discord_metadata_jsonb/up.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE "public"."guild" ADD COLUMN "discord_metadata" jsonb NULL;
+
+UPDATE "public"."guild" SET "discord_metadata" = ('{"inviteUrl": "' || discord_invite_url || '"}')::jsonb;
```

Comment:
I don't know how works jsonb in postgres, but what means `||` ? and isn't `discord_invite_url || '"` missing `:` ?

Reply:
`||` is SQL string concatenation.